docs/UserGuide.md

```diff
@@ -113,6 +113,7 @@ Parameters:
 </box>
 
 Restrictions:
+* Volunteers cannot have duplicate names(Volunteer names are unique to each volunteer).
```

Comment:
perhaps can add space between `names` and `(` so we don't get a `veryLow` documentation bug hahah